digitalearthau/config/products/wofs_apr_oct_summary.yaml

```diff
@@ -0,0 +1,37 @@
+name: wofs_apr_oct_summary
+description: Water Observations from Space April to October Statistics
+metadata_type: eo
+
+metadata:
+  product_type: wofs_apr_oct_summary
+  format:
+    name: 'NetCDF'
+  instrument:
+    name: 'TM,ETM,OLI'
+  platform:
+    code: 'LANDSAT_5,LANDSAT_7,LANDSAT_8'
```

Comment:
The `instrument` and `platform` sections probably aren't needed in the `metadata` section if they are a mix (as it is really a `dataset matching` section).  
Not really an issue until Landsat 9 is launched, but we can always code to make it understand lists of platforms, etc.